packages/esm-patient-registration-app/src/patient-registration/section/patient-relationships/relationships.resource.tsx

```diff
@@ -0,0 +1,70 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+import { FormValues } from '../../patient-registration-types';
+
+const customRepresentation =
+  'custom:(display,uuid,' +
+  'personA:(uuid,display,person:(age,display,birthdate,uuid)),' +
+  'personB:(uuid,display,person:(age,display,birthdate,uuid)),' +
+  'relationshipType:(uuid,display,description,aIsToB,bIsToA))';
+
+export async function getPatientRelationships(patientUuid: string): Promise<FormValues['relationships']> {
+  const response = await openmrsFetch<RelationshipsResponse>(
+    `/ws/rest/v1/relationship?v=${customRepresentation}&person=${patientUuid}`,
+  );
+  if (response.ok) {
+    const relationships = response?.data?.results;
+    return relationships.map((r) =>
+      r.personA.uuid === patientUuid
+        ? {
+            relatedPersonName: r.personB.person.display,
+            relatedPersonUuid: r.personB.person.uuid,
+            relation: r.relationshipType.bIsToA,
+            relationshipType: `${r.relationshipType.uuid}/bIsToA`,
```

Comment:
Instead of doing the `split` in the form component, why not just add a `direction` property?